deployment/cdk/opensearch-service-migration/lib/stack-composer.ts

```diff
@@ -53,7 +53,9 @@ export class StackComposer {
         const openAccessPolicyEnabled = getContextForType('openAccessPolicyEnabled', 'boolean')
         const availabilityZoneCount = getContextForType('availabilityZoneCount', 'number')
         const migrationAssistanceEnabled = getContextForType('migrationAssistanceEnabled', 'boolean')
-        const sourceCWLogStreamARN = getContextForType('sourceCWLogStreamARN', 'string')
+        const MSKARN = getContextForType('MSKARN', 'string')
```

Comment:
Could you add these three options to the Configuration Options table in the README. It will be helpful for others to be able to use

Reply:
Will do!